### ISSUE TITLE
[BUG] Segmentation Fault 11 on Modified oxView DNA Origami Topology

### ISSUE THREAD

**Issue by joecamxtc** [lammps/lammps#4341]
Hello,

I am using LAMMPS 2Aug-2023 on Mac OS M1 Pro, and am trying to extend the link bond (the bond that links each beam of my 2-beam device) of my structure such that I have several origami of varying length. However, I will receive a Segmentation Fault 11 when nothing changes except this linker length is increased.
![image](https://github.com/user-attachments/assets/ad1296e7-f03b-4043-93d9-fa0d636cd121)

An image of what I would like, as well as incrementally smaller and perhaps larger linkers.

I expected LAMMPS to output trajectory files of the equilibrated structure in the same way I received them using output10.dat.lammps.txt. What actually happens with this elongated data file is a segmentation fault with no further debuggable information.

This leads me to believe I have a GIGO problem, but I must take these steps of linker elongation to later do the analyses I need to do.

You can reproduce the issue if you like by using this script, replacing read_data file with this “output10_xt0.dat.lammps.txt” data file.

I was wondering if there were any “LAMMPS input tips” to make the simulation behave, such as the following which I’ve tried:

-changing the neighbor list skin distance

-“super-damping”: making my damp parameter lower than it already is, beyond 0.01

-changing the energy minimization parameters

In short, I don't understand why LAMMPS is able to handle my initial structure, output10.dat.lammps.txt, but cannot understand a slight modification to the structure (output10_xt0.dat.lammps.txt).
[output10.dat.lammps.txt](https://github.com/user-attachments/files/17152690/output10.dat.lammps.txt)
[output10xt0.dat.lammps.txt](https://github.com/user-attachments/files/17152734/output10xt0.dat.lammps.txt)
[input2.lammps (2).txt](https://github.com/user-attachments/files/17152777/input2.lammps.2.txt)

Thanks for any information or assistance you may be able to provide.

**Comment by ohenrich** [comment]
Could you please try to update to stable_29Aug2024 or add the flag `pair_modify neigh/trim no` to your input script. There has been a problem with the recently introduced default trim of neighbour lists, which we rectified in the latest stable release by setting a default flag that prevents trimming.

Let me know how you are getting on. I'm back 1st October.

**Comment by joecamxtc** [comment]
> Let me know how you are getting on. I'm back 1st October.


Still experiencing segmentation fault 11 on stable_29Aug2024 release. I tried the flag in both versions (after pair_style).

Is there any way to change the maximum bond length in bond_coeff without total reparameterization of the model? Maybe through xi or kappa?

**Comment by ohenrich** [comment]
@joecamxtc @akohlmey This is not a bug, so I removed the label. Please remove this issue from the LAMMPS Bug Reports.

The problem is triggered by the very frustrated input structures in your data files. I can run both structures when I omit some unsuitable commands following a standard MD protocol in your attempt to equilibrate the structures (see attached modified input file and two log files for both structures).

In fact, I can run with what I would call production parameters (`damp 2.5` in the `fix langevin`, `timestep 1e-2`). However, this could lead to some undesirable configurational changes as the initially large kinetic energy needs to dissipate first. Hence, I recommend another protocol to obtain equilibrated structures:

1. `damp 0.01, timestep 1e-5`, 1000 - 10000 iterations
2. `damp 0.01, timestep 1e-4`, 1000 - 10000 iterations
3. `damp 0.01, timestep 1e-3`, 1000 - 10000 iterations

This is basically overdamped Langevin dynamics with increasing timestep size. It should allow your structures to equilibrate and creep into place without blowing up.

[log1.lmp.txt](https://github.com/user-attachments/files/17192125/log1.lmp.txt)
[log2.lmp.txt](https://github.com/user-attachments/files/17192126/log2.lmp.txt)
[input2.lammps.2_modified.txt](https://github.com/user-attachments/files/17192127/input2.lammps.2_modified.txt)
